src/components/Header/style.ts

```diff
@@ -26,13 +26,14 @@ export const Inner = styled.div`
 `;
 
 export const Logo = styled.button`
+  display: flex;
+  align-items: center;
   color: #000;
-  leading-trim: both;
-  text-edge: cap;
   font-size: 1.3125rem;
   font-style: normal;
   font-weight: 700;
   line-height: normal;
+  column-gap: 0.5rem;
```

Comment:
```suggestion
  column-gap: 0.38rem;
```

figma 상으로 gap이 0.38rem으로 보입니다.
한번 더 확인해주시면 감사하겠습니다.

Reply:
다음부터는 잘확인하도록 하겠습니다